### ISSUE TITLE
Create drupal.org project

### ISSUE THREAD

**Issue by mrconnerton-encorehc** [FameHelsinki/search_api_elastic#1]
Is this module going to be released as a drupal.org project?

**Comment by KalleVuorjoki** [comment]
I don't think so, I'd prefer to have only single Elasticsearch codebase for Drupal. The reason for this repository is to investigate the possibility to use [drupal/search_api_opensearch](https://www.drupal.org/project/search_api_opensearch) as base for new branch for [drupal/elasticsearch_connector](https://www.drupal.org/project/elasticsearch_connector) module. The related drupal.org issue is [#3270464 ](https://www.drupal.org/project/elasticsearch_connector/issues/3270464), but in order to keep backward compatibility from 8.x-7.x --> 8.0.x upgrade, it would need the event subscribers to work. I reported that on [#3296449](https://www.drupal.org/project/search_api_opensearch/issues/3296449)
